core/deployment/pom.xml

```diff
@@ -158,6 +158,10 @@
             <scope>test</scope>
             <type>pom</type>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-spring-core-api</artifactId>
```

Comment:
might be an smell. but depending on some api seems the way to go choosing centralization over duplication.